src/sentry/static/sentry/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -232,13 +232,13 @@ export default class AbstractExternalIssueForm<
       {}
     );
 
+    const {Header, Body} = this.props as ModalRenderProps;
```

Comment:
do we need Type casting here?

Reply:
I'm actually not quite sure why it needs this. but it indeed does. Maybe I am missing something obvious.

Reply:
yeah maybe we should double-check it

Reply:
I unfortunately can't seem to find a way around this. I'm sure there's something going on with the way the props are extended. But I'm just not going to worry about this.